index.bs

```diff
@@ -293,18 +298,29 @@ When the <dfn method for="XR">requestSession(|mode|)</dfn> method is invoked, th
   1. Let |immersive| be <code>true</code> if |mode| is {{XRSessionMode/"immersive-vr"}} or {{XRSessionMode/"immersive-ar"}}, and <code>false</code> otherwise.
   1. If |immersive| is <code>true</code>: 
     1. If [=pending immersive session=] is <code>true</code> or [=active immersive session=] is not <code>null</code>, [=reject=] |promise| with an "{{InvalidStateError}}" {{DOMException}} and abort these steps.
-    1. Else set [=pending immersive session=] to be <code>true</code>.
+    1. Set [=pending immersive session=] to be <code>true</code>.
   1. [=ensures an XR device is selected|Ensure an XR device is selected=].
-  1. If the [=XR/XR device=] is null, [=reject=] |promise| with null.
-  1. Else if the [=XR/XR device=]'s [=list of supported modes=] does not [=list/contain=] |mode|, [=reject=] |promise| with a "{{NotSupportedError}}" {{DOMException}}.
-  1. Else if |immersive| is <code>true</code> and the algorithm is not [=triggered by user activation=], [=reject=] |promise| with a "{{SecurityError}}" {{DOMException}}.
+  1. Follow the steps for the first matching conditions:
+      <dl class="switch">
+      <dt> If the [=XR/XR device=] is <code>null</code>
+      <dd> [=Reject=] |promise| with <code>null</code>.
+      <dt> Else if the [=XR/XR device=]'s [=list of supported modes=] does not [=list/contain=] |mode|
+      <dd> [=Reject=] |promise| with a "{{NotSupportedError}}" {{DOMException}}.
+      <dt> Else if |immersive| is <code>true</code> and the algorithm is not [=triggered by user activation=]
+      <dd> [=Reject=] |promise| with a "{{SecurityError}}" {{DOMException}}.
+    </dl>
   1. If |promise| was [=rejected=]:
     1. If |immersive| is <code>true</code>, set [=pending immersive session=] to <code>false</code>.
     1. Abort these steps.
   1. Let |session| be a new {{XRSession}} object.
   1. [=Initialize the session=] with |session| and |mode|.
-  1. If |immersive| is <code>true</code>, set the [=active immersive session=] to |session|, and set [=pending immersive session=] to <code>false</code>.
-  1. Else append |session| to the [=list of inline sessions=].
+  1. Potentially set the [=active immersive session=] based on the following:
+    <dl class="switch">
+      <dt> If |immersive| is <code>true</code>
+      <dd> Set the [=active immersive session=] to |session|, and set [=pending immersive session=] to <code>false</code>.
+      <dt> Else
```

Comment:
Line 244 uses Otherwise, which is also used in at least some other specs. Most importantly, this spec should be consistent.